code/game/machinery/slotmachine.dm

```diff
@@ -334,7 +335,8 @@
 			H.throw_at(target, 3, 10)
 	else
 		var/value = coinvalues["[cointype]"]
-
+		if(value <= 0)
+			CRASH("Coin value of zero, refusing to payout in dispenser")
```

Comment:
Change it to insert the actual value of the coin.  Knowing if it's negative would help with future debugging, should someone somehow royally mess up.

Reply:
no